webpack.common.js

```diff
@@ -16,7 +16,7 @@ module.exports = {
     rules: [
       {
         test: /\.tsx?$/,
-        exclude: /node_modules/,
+        exclude: /node_modules\/(?!(\@api)\/).*/,
```

Comment:
this actually causes the contents of `/node_modules/@api/*` to be transpiled by the specific loader (`ts-loader`), which is not correct as contents of packages should not be transpiled. perhaps you are looking for something like [`watchOptions`](https://webpack.js.org/configuration/watch/)?

Reply:
You're right. Awesome! Will revert this change.